InvenTree/InvenTree/version.py

```diff
@@ -130,3 +130,39 @@ def inventreeCommitDate():
 
     commit_dt = dt.fromtimestamp(main_commit.commit_time) + td(seconds=main_commit.commit_timezone)
     return str(commit_dt.date())
+
+
+def inventreeInstaller():
+    """Returns the installer for the running codebase - if set."""
+    # First look in the environment variables, e.g. if running in docker
+
+    installer = os.environ.get('INVENTREE_PKG_INSTALLER', '')
```

Comment:
Why aren't you using None as default and return that, it's mich shorter?

Reply:
Because the PR is not done yet and there is more to come